commands/runserver/runserver.drush.inc

```diff
@@ -131,11 +131,15 @@ function drush_core_runserver($uri = NULL) {
     drush_start_browser($browse, 2);
   }
   // Start the server using 'php -S'.
-  if (drush_drupal_major_version() >=8) {
-    $extra = ' "' . __DIR__ . '/d8-rs-router.php"';
+  if (drush_drupal_major_version() >= 8) {
+    $extra = ' "' . __DIR__ . '/rs-router.php"';
+  }
+  elseif (drush_drupal_major_version() == 7) {
+    $extra = ' "' . __DIR__ . '/rs-router_7.php"';
   }
   else {
-    $extra = ' --define auto_prepend_file="' . __DIR__ . '/runserver-prepend.php"';
+    // For Drupal 6 we prepend an include to what PHP was going invoke.
```

Comment:
The master branch of Drush no longer supports Drupal 6. I dont mind removing this bit at merge time if that makes things simple.